test/REGRESSIONS

```diff
@@ -686,6 +679,668 @@ Reviewed 2015-01-23
 ===========================
 
 
+===============================
+cray-module-correctness-cray-x?
+Inherits 'general'
```

Comment:
Intuitively, it seems to me that there should be some sort of inheritance relationship between the various cray-module-correctness testing configurations and the whitebox testing rather than making cray-module-correctness testing inherit directly from 'general', but I'm not certain.  I'm curious what @thomasvandoren thinks about this.